pyre.py

```diff
@@ -47,9 +51,46 @@ def color(r, g, b):
     assert(len(self.particles) == self.NUM_PARTICLES)
 
     self.resize()
+    self.volume = 1.0
+    if pygame_available:
+      mixer.init()
+      music.load('fire.wav')
+      music.play(-1)
+    elif pygame_available == False and pyaudio_available:
```

Comment:
`elif` implies `pygame_available == False`, perhaps make this line
`elif pyaudio_available:` for conciseness